_plugins/genfeatures.rb

```diff
@@ -79,7 +87,13 @@ def generate_from_section(site, section, timestamp, category, appended_title = "
 					"windows" => {
 					"*" => "u"
 					}
-				}
+				},
+				"chrome_android" => {
```

Comment:
You can open a follow up issue if this is a faf to do but I think we should rather track the regular BCD features separately from the WebView features long term. It helps for mapping and comparing today but this means canIwebview has no way to know that the titular WebViews are more important than the browsers. This will matter for baseline generation etc.

Maybe we can just have a hard coded field called something like "is_webview"?

Reply:
I'll change the nicenames to "Chrome Browser" and "Safari Browser" for now. And the are ordered to the end. But let's find a good solution in #32.